web-app/src/router/routes.ts

```diff
@@ -6,10 +6,15 @@ const routes: RouteConfig[] = [
     component: () => import('layouts/MainLayout.vue'),
     children: [
       {
-        path: '',
+        path: '/index',
```

Comment:
```suggestion
        path: '',
```
Under `/app` or `/app/tournament(s)` path we should have a tournament list view
`/app/tournament/:id` - details
`/app/tournament/creator` - creator